src/main/java/hoshi/task/Deadline.java

```diff
@@ -0,0 +1,62 @@
+package hoshi.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class represents a Deadline with a description, completion status and end time.
+ *
+ * <p>This class inherits from the Task class It maintains a description
+ * of the deadline, whether the task has been completed or not and the end time.</p>
+ */
+
+public class Deadline extends Task {
+
+    private LocalDate endTime;
```

Comment:
I think its a bit misleading to call it endTime as the type is dealing with Dates not DateTimes. EndDate would be better naming wise in my opinion.